templates/_base/layout.twig

```diff
@@ -7,19 +7,20 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{{ block('title')|striptags }}</title>
     {% block stylesheets %}
-    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600" rel="stylesheet">
```

Comment:
Are you sure '600' exists?

Reply:
Ah, yes.. `600` exists.. Much better! 👍

Reply:
Yeah 700 was too heavy so I have set bold to be semi bold